docs/reference/authorization.md

```diff
@@ -52,8 +52,12 @@ When a user makes a request to perform that action, the following process takes
 Each policy attached to a user or a group has an `Effect` - either `allow` or `deny`.
 During evaluation of a request, `deny` would take precedence over any other `allow` policy.
 
-This helps us compose policies together. For example, we could attach a very permissive policy to a user and use `deny` rules to then selectively restrict what that user can do.
+If no permissions exist than the action will be blocked, making no permission the equivilent of `deny`. 
```

Comment:
This is rather confusing to me, although I understand why it is correct. Deny is an explicit operation, whereas no permissions is a default. They are not equivalent: for instance, adding a single Allow permission yields a different result from each. They do have equal effects.
Perhaps 
```suggestion
The default if no permissions exist is to allow nothing.
```